test/cli_test.py

```diff
@@ -518,14 +505,14 @@ def test_convert_write_tags(self):
 
         # We "convert" by copying the file. Setting the title simulates
         # a badly behaved converter
-        mediafile_converted = MediaFile(syspath(item.path))
+        mediafile_converted = MediaFile(item.path)
```

Comment:
`item.path` should be the beets-internal representation (i.e. `bytes`), right? Then, from my understanding, the `syspath` is required as that represenation is not the correct one to pass to OS APIs on Windows.

Reply:
I don’t know enough about `MediaFile` to understand what’s going on. But this is test code and it works on both platforms on CI some I’m ok with just using it as is.